index.js

```diff
@@ -77,8 +77,9 @@ function extractJSONLDfromHTML(url, html) {
  * If dataCatalogUrl is not supplied, the default OA Data Catalog (https://openactive.io/data-catalogs/data-catalog-collection.jsonld) is used.
  *
  * @param {string} [dataCatalogUrl]
+ * @param {boolean} [suppressConsoleErrors]
  */
-async function getAllDatasets(dataCatalogUrl = 'https://openactive.io/data-catalogs/data-catalog-collection.jsonld') {
+async function getAllDatasets(dataCatalogUrl = 'https://openactive.io/data-catalogs/data-catalog-collection.jsonld', suppressConsoleErrors = false) {
```

Comment:
Does this new param need to be documented in the readme? Is it best practice to include console errors in a library, or is it better just to throw an error?

Reply:
We can't throw an error here because so many dataset site fail. Best case I think is allow possible suppression of the errors, but the default is non-suppressed so it's clear which have failed.

Reply:
Maybe it could return a "status report" object kinda like validator which includes a list of dataset sites that have failed?

Reply:
Added `errors`